hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -320,4 +336,35 @@ def submit_post_url(
             message="Failed to generate upload url",
         )
 
+    @submit_api.post("/hash/", apply=[jsoninator(SubmitContentHashRequestBody)])
+    def submit_hash(
+        request: SubmitContentHashRequestBody,
+    ) -> t.Union[SubmitResponse, SubmitError]:
+        """
+        Endpoint to submit a hash of a piece of content
+        """
+
+        # Record content object (even though we don't store anything just like with url)
+        _record_content_submission_from_request(request)
+
+        # Record hash
+        # todo add MD5 support and branch based on request.hash_type
+        #   note: quality of PDQ hashes should part of `signal_specific_attributes` after #749/related is merged
+        hash_record = PipelineHashRecord(
+            content_id=request.content_id,
+            signal_type=PdqSignal,
+            content_hash=request.signal_value,
+            updated_at=datetime.datetime.now(),
+        )
+        hash_record.write_to_table(dynamodb_table)
+
+        # Send hash directly to matcher
+        # todo this could maybe try and reuse the methods in UnifiedHasher in #749
+        _get_sqs_client().send_message(
+            QueueUrl=pdq_hash_queue_url,
+            MessageBody=json.dumps(hash_record.to_legacy_sqs_message()),
+        )
```

Comment:
I am almost certain you can submit to the hashes_eueue instead of pdq_hashes_queue. 

The unified matcher supports PDQ indexes already. What remains to be migrated is s3 events -> unified hasher.

Reply:
done and retested, looks like that works!